dm/dm/ctl/master/operate_source.go

```diff
@@ -136,6 +136,9 @@ func operateSourceFunc(cmd *cobra.Command, _ []string) error {
 			}
 			content = []byte(yamlStr)
 		}
+		if cfg.RelayDir != "" {
+			common.PrintLinesf("`relay-dir` in source config will be deprecated soon, please use `relay-dir` in worker config instead")
```

Comment:
do we also add this warnning to openapi ?

Reply:
cc @sunzhaoyang

Reply:
No need. We should describe this information in the openapi's documents .